docs/index.rst

```diff
@@ -40,26 +34,43 @@ To install from Anaconda, the NREL channel needs to be appended::
     conda config --env --append channels nrel
     conda install nrel-pysam
 
-Each technology and financial model is its own module, which can be imported by::
+Writing Your First PySAM Script
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-	import PySAM.<PySAM Module>
+Before you write your first PySAM script it really helps to familiarize 
+yourself with SAM.  If you are not already familiar with SAM, it is recommended
+you install it, and run a simulation of interest to you.  SAM is a graphical user interface to the SAM Simulation Core library of models.  Using it provides an easy introduction to the simulations that can be done using the library.  PySAM is a python interface to the same SAM Simulation Core that allows you to customize SAM for your specific needs.
 
-The original Python SDK, PySSC, is bundled, and can be imported by::
+There is a lot of data needed to run a SAM simulation.  Entering that data with assignment statements in PySAM can be tedious.  Fortunately, you can enter the data using the SAM graphical user interface, and save that data to a JSON file, which can then be loaded by PySAM. You can then modify and use the data in PySAM with python as needed.
```

Comment:
I think you could also mention the potential to create a default simulation as well to get around all the data needed.